examples/fungible-token/tests/workspaces.rs

```diff
@@ -190,22 +188,22 @@ async fn simulate_transfer_call_with_burned_amount() -> anyhow::Result<()> {
     let (contract, _, defi_contract) = init(&worker, initial_balance).await?;
 
     // defi contract must be registered as a FT account
-    register_user(&worker, &contract, defi_contract.id()).await?;
+    register_user(&contract, defi_contract.id()).await?;
 
     // root invests in defi by calling `ft_transfer_call`
     // TODO: Put two actions below into a batched transaction once workspaces supports them
```

Comment:
This seems to be the last TODO left, should we handle it in this PR and be done with it?